src/utils/api/pm/index.tsx

```diff
@@ -60,7 +60,7 @@ const getBpayList = async (
     (response) =>
       api.utils.matchApiStatus(response, () => {
         const { value, status } = response;
-        if (status === 200 && value?.data && !!value.data.length) {
```

Comment:
I think that by Adding this check when the array is empty an instance of E.left is returned. As result the new OUTCOME_ROUTE.ACCOUNT_BPAY_NOT_PRESENT outcome will never happen. Maybe you should consider to remove the last one check

Reply:
That's the old version :)
Also I made the check stricter in 4fc4525, what do you think?

Reply:
I think that it's not necessary to add this last check :)
As a test you could run the unit tests on 46f42 before and after removing the check and see that they will pass anywhere.

Reply:
`IBpayAccountItems` cannot be `undefined` so the check must be made or it doens't type check

Reply:
We should also adjust the function description removing "returns an instance of O.none when no accounts are returned"

Reply:
Fixed in 424f77c